docs/release-notes/s3gw-v0.15.0.md

```diff
@@ -0,0 +1,39 @@
+# Release Notes - v0.15.0
+
+This release focuses on stabilizing our continuous integration and release process.
+In this context, we have also addressed a number of issues that was affecting our
+testing framework when automatically triggered by CI.
+
+This activity, although not introducing a direct improvement consumable by the user,
```

Comment:
"Although this activity may not result in any direct user-facing improvements, it plays a crucial role in maintaining a stable environment for the upcoming major enhancements that the s3gw team is currently developing."